utils/build/docker/dotnet_http/parametric/Endpoints/ApmTestApi.cs

```diff
@@ -0,0 +1,325 @@
+﻿using Datadog.Trace;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Net.Http.Headers;
+using Newtonsoft.Json;
+
+namespace ApmTestApi.Endpoints;
+
+public abstract class ApmTestApi
+{
+    public static void MapApmTraceEndpoints(WebApplication app, ILogger<ApmTestApi> logger)
+    {
+        _logger = logger;
+        _ = Tracer.Instance;
```

Comment:
I was wondering why this was here and there is a comment on the other project that I think should be copied over

```
            // TODO: Remove when the Tracer sets the correct results in the SpanContextPropagator.Instance getter
            // This should avoid a bug in the SpanContextPropagator.Instance getter where it is populated WITHOUT consulting the TracerSettings.
            // By instantiating the Tracer first, that faulty getter code path will not be invoked
```